apps/storybook-e2e/src/e2e/elements/multiselect/vl-multiselect.stories.cy.ts

```diff
@@ -0,0 +1,8 @@
+const multiSelectUrl = 'http://localhost:8080/iframe.html?id=elements-multiselect--multiselect-default';
+
+describe('story vl-multiselect - default', () => {
+    it('should display story', () => {
+        cy.visit(multiSelectUrl);
+        cy.get('select[is="vl-multiselect"]').select('Frankrijk').should('have.value', 'Frankrijk');
```

Comment:
Dit gaat idd testen of de `select[is="vl-multiselect"]` value `Frankrijk` heeft; maar dit gaat even goed werken op een native `select` dus dit geeft geen zekerheid dat de `vl-multiselect` web component geregistreerd is.

Reply:
deze test faalde trouwens, ik wijzigde het naar
cy.get('select[is="vl-multiselect"]').should('have.attr', 'data-vl-select-dressed', 'true');
dat is een attribuut van ons dat door de webcomponent toegevoegd wordt